overlore/townhall/logic.py

```diff
@@ -19,18 +21,23 @@ def get_townhall_summary(townhall: str) -> tuple[str, str]:
     return (res[0], res[1])
 
 
-async def handle_townhall_request(message: str, config: Config) -> str:
+async def handle_townhall_request(message: str, config: Config) -> tuple[int, str, str, str]:
     events_db = EventsDatabase.instance()
     vector_db = VectorDatabase.instance()
     gpt_interface = OpenAIHandler.instance()
+    realms = Realms.instance()
 
     data = str_to_json(message)
 
     realm_id = int(data.get("realm_id"))
+    realm_name = realms.name_by_id(realm_id)
+
+    realm_order_id = int(data.get("order"))
+    realm_order = realms.order_by_order_id(realm_order_id)
```

Comment:
There is a way to receive the order of the realm sending the message. Might be wiser to use this, as it uses the actual data instead of copying the order ids in our backend, and would prevent errors in case of them getting changed in the client side

Reply:
This is unlikely that it will change as it's a property of the NFTs. But agreed, will raise an issue about this